set-cuda-path/README.md

```diff
@@ -0,0 +1,28 @@
+# set-cuda-path
+
+This action set the cuda path into GITHUB_ENV amd GITHUB_PATH.
+
+## Usage
+
+```yaml
+jobs:
+  set-cuda-path:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set cuda path
+        id: set-cuda-path
+        uses: autowarefoundation/autoware-github-actions/set-cuda-path@v1
+```
```

Comment:
[nits] I think using `CUDA` is better in normal sentences.

```suggestion
This action set the CUDA path into GITHUB_ENV amd GITHUB_PATH.

## Usage

```yaml
jobs:
  set-cuda-path:
    runs-on: ubuntu-latest
    steps:
      - name: Check out repository
        uses: actions/checkout@v3
        with:
          fetch-depth: 0

      - name: Set CUDA path
        id: set-cuda-path
        uses: autowarefoundation/autoware-github-actions/set-cuda-path@v1
```
```